content/blog/announce-tonic-0-1.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Tonic: gRPC has come to async/await!"
+url: "/tonic-grpc-has-come-to-async-await"
+date: 2019-10-01T21:00:00-04:00
+draft: false
+---
+
+# Tonic: gRPC has come to async/await!
+
+I am pleased to finally announce a crate that I have been working very hard on for the
+past few months. [`tonic`] has finally hit the initial [`0.1.0-alpha.1`] release! Stable
+releases will follow in the coming months.
+
+### What is Tonic?
+
+> Tonic is a gRPC-over-HTTP/2 implementation focused on high performance, interoperability, and flexibility. This library was created to have first class support of async/await and to act as a core building block for production systems written in Rust.
+
+Tonic began its life as  [`tower-grpc`], which was built to satisfy [linkerd]'s need for a production-ready gRPC implementation. [`tower-grpc`] was based upon the [`tower`] library, which has production uses in systems such as the [`linkerd-proxy`], [`vector`], and the [`noria`] database at MIT. The [`tower`] library has also seen usage in additional libraries such as  [`warp`], [`tower-web`], and [`actix-net`].
+
+With async/await’s forthcoming stabilization, we created Tonic to support the new syntax natively. This means clients will support async/await out of the box and server
+implementations can be defined via [`async_trait`]s. This provides an unparalleled experience for
+writing async services quickly and efficiently. Not only does Tonic provide a fully featured gRPC implementation but it also comes with a fully featured, batteries-included, HTTP/2 client and server built
+around [`hyper`], [`tokio`]and [`tower`]. Both the client and server implementation provide TLS backed by
+either [`openssl`] or [`rustls`]. The client provides load balancing, interceptors, timeouts, rate limiting,
+concurrency control and more!
+
+Tonic also boasts strong interoperability and correctness. Every commit is checked that it passes
+the [gRPC interop test cases][interop-cases] against the [`grpc-go`] implementation. Hopefully, more
+languages can be added in the future.
+
+### Features
+
+Out of the gate, the goal for Tonic has been to provide a good batteries included experience. It
+already supports many features, with many more planned! Here is a list of features:
+
+- Implemented in pure Rust (minus [`openssl`] which is optional)
+- Interoperability tested via [`tonic-interop`]
+- Bi-directional streaming
+- Custom metadata
+- Trailing metadata
+- Codegen via [`prost`]
+- Exposes [`tracing`] diagnostics (more to come)
+- Fully featured HTTP/2 client and server
+- TLS backed by either [`openssl`] or [`rustls`]
+- Load balancing powered by [`tower`]
+- Reliability features such as timeouts, rate limiting, concurrency control, and more
```

Comment:
"Timeouts, rate limiting, concurrency control, and a growing set of other reliability-focused tools" ?